doccano_client/beta/controllers/span.py

```diff
@@ -68,7 +68,7 @@ def create(self, span: Span) -> SpanController:
         """Create a new span, return the generated controller
 
         Args:
-            span: Span. The only fields that will be uploaded are text, annnotations, and meta.
+            span: Span. See mock_api_responses for the desired format.
```

Comment:
Here as well. If it's not too much to ask might good to add the format here quickly, saving future contributors/readers of code from having to look at another file.